.babelrc.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  presets: [
+    [
+      '@babel/preset-env',
+      process.env['BABEL_ENV'] === 'es6'
+        ? {
+            modules: false,
+          }
+        : {},
+    ],
+    '@babel/preset-react',
+  ],
+  plugins: [
+    '@babel/plugin-transform-proto-to-assign',
+    '@babel/plugin-transform-flow-strip-types',
+    'lodash',
+    '@babel/plugin-transform-runtime',
+    'transform-remove-console',
+    'transform-remove-debugger',
+    'transform-minify-booleans',
+    '@babel/plugin-syntax-dynamic-import',
+    '@babel/plugin-syntax-import-meta',
+    '@babel/plugin-proposal-class-properties',
+    '@babel/plugin-proposal-json-strings',
+    [
+      '@babel/plugin-proposal-decorators',
```

Comment:
Why are we adding all these babel plugins? I don't think we are using decorators syntax in this repository, are we?

Reply:
I'd think we should be able to significantly simplify the transpilation setup for this package. Just use some babel presets

Reply:
Well, the whole idea of babel 7 is to remove the `stage-x` or the yearly presets in favor of explicitly using the individual proposals. See:

https://babeljs.io/docs/en/v7-migration#yearly-preset-deprecations-blog-2017-12-27-nearing-the-70-releasehtml-deprecated-yearly-presets-eg-babel-preset-es20xx

So far, the 6->7 migration automatically added all plugins that were included in "es2015", "stage-2", "react" that we used before.  
But yes, we surely can remove some of them that are not in use.